voluncheer/jobboard/apps.py

```diff
@@ -4,4 +4,4 @@
 class JobBoardConfig(AppConfig):
```

Comment:
If jobboard is one word all references to it when using PascalCase should be `Jobboard` and in snake_case `jobboard`.